src/SequenceManager.cc

```diff
@@ -358,4 +397,56 @@ void SequenceManager::store_sequence(const Sequence& seq) const
         store_step(seq_path / extract_filename_step(++idx, max_digits, step), step);
 }
 
+std::string SequenceManager::stage_files_in_directory(std::filesystem::path dir_name, const std::string& filetype)
+{
+
+    // detect what has changed in the sequence
+    std::string git_msg{""};
+    auto stats = gl_.status();
+    for(const auto& elm: stats)
+    {
+        
+        // filter for changes in sequence
+        if (gul14::starts_with(elm.path_name, (std::string) dir_name))
+        {
+            if (elm.changes == "new file" and (filetype == elm.changes or filetype == ""))
+            {
+                git_msg += gul14::cat("\n", "- create '",elm.path_name, "'");
+            }
+            else if (elm.changes == "modified" and (filetype == elm.changes or filetype == ""))
+            {
+                git_msg += gul14::cat("\n", "- modify '",elm.path_name, "'");
+            }
+            //TODO: file number is changing. Figure out which file is deleted
+            //INFO: Maybe git figures it out on its own and tag it with "renamed"
```

Comment:
??? Needs to be investigate by you?

Reply:
Yes, it is just a matter of description in the commit message. The changes will be stored nontheless, so I considered a detail which can be solved later (? maybe to be discussed next wednesday). I have to add a test case for the removal of a step and the resulting change of enumeration of the files.

Reply:
I also have trouble reading and understanding these two comment lines. Could you reformulate them? And: If it is something we can take care of in the next week, there shouldn't be a TODO, but an implementation. If it is something that we cannot take care of in the next four or five weeks, there should not be a TODO either, because it will never be addressed.

Reply:
That code is gone.